pkg/services/screenshot/screenshot.go

```diff
@@ -322,12 +322,12 @@ func NewUploadingScreenshotService(r prometheus.Registerer, service ScreenshotSe
 		service:  service,
 		uploader: uploader,
 		uploadFailures: promauto.With(r).NewCounter(prometheus.CounterOpts{
-			Name:      "upload_failures",
+			Name:      "upload_failures_total",
 			Namespace: namespace,
 			Subsystem: subsystem,
 		}),
 		uploadSuccesses: promauto.With(r).NewCounter(prometheus.CounterOpts{
-			Name:      "upload_successes",
+			Name:      "upload_successes_total",
```

Comment:
```suggestion
			Name:      "upload_success_total",
```

typically named in present tense

Reply:
I have `cache_hits_total` and `cache_misses_total`, should I fix these too? Something like `cache_hit_total` and `cache_miss_total`?

Reply:
The justification for `successes` rather than `success` was the example `http_requests_total` rather than `http_request_total`.

Reply:
Looking at some of our metrics as examples, for cache's `misses_total` seems to be a majority (with coredns, redis and etcd as examples) whoever for `success` it seems that `success` is more common than `successes` with very hits (only 1 occurrence across _all_ of our metrics).

I'll keep your cache metric names as is (adding the area prefix if they don't have them) and just change this one.